vernac/prettyp.ml

```diff
@@ -323,15 +323,19 @@ let print_arguments ref =
     | Some (UnfoldWhen { nargs; recargs }) -> [], recargs, nargs
     | Some (UnfoldWhenNoMatch { nargs; recargs }) -> [`ReductionDontExposeCase], recargs, nargs
   in
-  let renames = try Arguments_renaming.arguments_names ref with Not_found -> [] in
+  let names, not_renamed =
+    try Arguments_renaming.arguments_names ref, false
+    with Not_found ->
+      let ty, _ = Typeops.type_of_global_in_context (Global.env()) ref in
+      Impargs.compute_implicits_names (Global.env()) Evd.empty (EConstr.of_constr ty), true in
```

Comment:
Please don't use evd.empty (from_env is better)

Reply:
You're right...